test/passing/refs.default/js_source.ml.ref

```diff
@@ -712,7 +712,7 @@ let _ =
   fooooooooooooooooooooooooooooooo
   |> fooooooooooooooooooooooooooooooo ~fooooooooooooooooooooooooooooooo
        ~fooooooooooooooooooooooooooooooo:(fun foo ->
-         match bar with Some _ -> foo | None -> baz)
+      match bar with Some _ -> foo | None -> baz)
```

Comment:
this is unfortunate, but in the spec: the body of the function is indented 4 characters from `|>`

Reply:
It's now fixed, at the price of a little ugliness when using `max-indent=2`, but that ugliness was already existing when doing `map li ~f:(fun` without `|>`.=, so its consistent.